src/utils/validateName.ts

```diff
@@ -15,7 +15,8 @@ const URL_REGEX =
   /^((?:(?:http?|ftp)[s]*:\/\/)?[a-z0-9-%\/\&=?\.]+\.[a-z]{2,4}\/?([^\s<>\#%"\,\{\}\\|\\\^\[\]`]+)?)/;
 const WHITE_SPACE_REGEX = /^\S*$/;
 const NUM_REGEX = /^[1-9](\d?|\d+)$/;
-const MNEMONICS_LENGTH_REGEX= /^([a-zA-Z0-9]+\s?){25}$/;
+const MNEMONICS_LENGTH_REGEX= /^(?:\s*[a-zA-Z]+(?:\s+[a-zA-Z]+){24})?\s*$/;
```

Comment:
you can use `\w` meta for `a-zA-Z0-9A-Z`, but I assume you want to exclude 0-9, `^([a-zA-Z_]+\s?){25}$` this should work.

Reply:
^([a-zA-Z_]+\s?){25}$ this regex accepts less that 25 words, as it's counting whitespaces as words.